lips/lip-20.md

```diff
@@ -0,0 +1,426 @@
+system. Details on these operations and how they effect the `total_value` for the currency
+are given in the section on [creation and removal of value](#minting_and_burning) from the
+system.
+
+### `preburn_value`
+
+This field holds the value of all coins currently slated for removal from the system.
+
+#### Mutability
+
+This field's value remains constant with the exception of removal operations as detailed in the section
+on [creation and removal of value](#minting_and_burning) from the system.
+
+### `can_mint`
+
+This specifies whether additional value in the specified currency may be added to the
+system. The `can_mint` field of a currency being set to `false` does not effect any other
+aspect of the system with respect to the given currency other than disallowing the
+introduction of more value in the given currency to the system. In particular, coins can
+be removed from the system, accounts may hold balances in the currency, and payments can
+be made in the currency.
+
+#### Mutability
+
+This field is only updatable through the
+[`Libra::update_minting_ability`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-update_minting_ability)
+function, and must be done by an account with the [Treasury Compliance role](https://github.com/libra/lip/blob/master/lips/lip-2.md#roles).
+
+## Capabilities Created at Registration
+
+Whenever a type `C` is registered as a currency, a
+[`MintCapability<C>`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#resource-mintcapability)
+and
+[`BurnCapability<C>`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#resource-burncapability),
+are created. If the currency is registered using the
+[`Libra::register_SCS_currency`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-register_scs_currency)
+function, both of these capabilities are stored under the Treasury Compliance account at address
+`0xB1E55ED`. In the case that the [`Libra::register_currency`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-register_currency)
+is used to register the currency both of these capabilities are returned to the caller, which may store them in other structures.
+The `MintCapability` and `BurnCapability` for each currency are [unique](https://github.com/libra/lip/blob/master/lips/lip-2.md#permissions); after
+registration of a currency no future mint and burn capabilities for the currency may be created.
+
+## On-chain list of registered currencies
+
+Whenever a currency is registered on-chain, the to-be-registered currencies currency code is added to the
+[`RegisteredCurrencies` on-chain config](https://github.com/libra/lip/blob/master/lips/lip-6.md#registered-currencies).
+The set of currency codes registered on-chain is always a set. Attempting to
```

Comment:
"the set... is always a set": there must be a better way to say that!

Reply:
It can't be a false statement though! :p 

I'll update.